app/Config/Events.php

```diff
@@ -46,7 +46,7 @@
 	 * --------------------------------------------------------------------
 	 * If you delete, they will no longer be collected.
 	 */
-	if (CI_DEBUG)
+	if (CI_DEBUG && ! is_cli())
```

Comment:
if I will use `php spark serve` then is `is_cli()` result to `true`? If so, does this mean we'll specifically disallow toolbar on development server?

Reply:
While I don't think so, more importantly this is the exact boolean construct already used in `Toolbar` itself, so we aren't changing any scenarios just disabling it sooner.